diff -up nana-1.7.4/build/cmake/enable_audio.cmake.omv~ nana-1.7.4/build/cmake/enable_audio.cmake
--- nana-1.7.4/build/cmake/enable_audio.cmake.omv~	2020-08-01 04:48:38.624070975 +0200
+++ nana-1.7.4/build/cmake/enable_audio.cmake	2020-08-01 04:52:21.619676131 +0200
@@ -4,12 +4,8 @@ option(NANA_CMAKE_ENABLE_AUDIO "Enable c
 if(NANA_CMAKE_ENABLE_AUDIO)
     target_compile_definitions(nana PUBLIC NANA_ENABLE_AUDIO)
     if(UNIX)
-        find_package(ASOUND) # ? https://github.com/hintjens/demidi/blob/master/Findasound.cmake
-        if(ASOUND_FOUND)
-            target_include_directories(nana PUBLIC ${ASOUND_INCLUDE_DIRS})
-            target_link_libraries(nana PUBLIC  ${ASOUND_LIBRARIES})
-        else()
-            message(FATAL_ERROR "libasound is not found")
-        endif()
+	#find_package(ASOUND) # ? https://github.com/hintjens/demidi/blob/master/Findasound.cmake
+	pkg_check_modules(ALSA "alsa")
+	target_link_libraries(nana PUBLIC ${pkgcfg_lib_ALSA_asound})
     endif()
-endif()
\ No newline at end of file
+endif()
